decisions/Gondor.txt

```diff
@@ -1785,6 +1785,23 @@ political_decisions = {
         effect = {
             prestige = 10
             badboy = 3
+			random_owned = {
+				limit = {
+					owner = {
+						HAR = {
+							exists = yes
+							vassal_of = GON
+						}
+					}
+				}
```

Comment:
Probably should include a check to see if HAR owns HAR_286

Reply:
Added it and unfucked the encoding